src/test/scala/chiselTests/stage/ChiselStageSpec.scala

```diff
@@ -40,7 +48,12 @@ class ChiselStageSpec extends AnyFlatSpec with Matchers with Utils {
   behavior of "ChiselStage.emitFirrtl"
 
   it should "return a High FIRRTL string" in {
-    ChiselStage.emitFirrtl(new Foo) should include ("mem bar")
+    ChiselStage.emitFirrtl(new Foo) should include ("mem memory")
+  }
+
+  it should "return a flattened FIRRTL string with '-e high'" in {
+    (new ChiselStage)
+      .emitFirrtl(new Foo, Array("-e", "high", "-td", "test_run_dir/ChiselStageSpec")) should include ("module Bar")
```

Comment:
I'd rather this not hardcode `test_run_dir`

Reply:
Switched to `createTestDirectory`. I've hardcoded this a lot in the past... 😬

Reply:
I'm watching you 🕵️